### DISCUSSION TITLE
Modal - padding-right

### DISCUSSION THREAD

**Discussion by diegok350** [saadeghi/daisyui#3173] [General]
Is there a property to set to remove the white space that appears when opening the modal, because it moves the elements on the screen?

![image](https://github.com/user-attachments/assets/c82840b8-042d-4af8-937c-77ff7f85e4f3)

**Comment by diegok350** [comment]
Solution:
To solve this, I added a custom CSS rule that leverages the :root pseudo-class combined with the :has() function. This rule ensures that when a modal is open, the scrollbar behavior is controlled, preventing the layout from shifting.

Here’s the CSS that fixed the issue:
:root:has(:is(.modal-open, .modal:target, .modal-toggle:checked + .modal, .modal[open])) {
    scrollbar-gutter: auto !important;
}

**Reply by GreyNewfie** [reply]
Thank you @diegok350 for posting this! I've been trying to solve a very similar issue and this worked. I had almost given up that I was going to find a solution.
